server/base/src/main/java/org/apache/accumulo/server/master/recovery/RecoveryPath.java

```diff
@@ -41,15 +50,46 @@ public static Path getRecoveryPath(Path walPath) {
 
       // drop wal
       walPath = walPath.getParent();
+      walPath = new Path(walPath, FileType.RECOVERY.getDirectory() + "-" + convertToSha1(portHost));
```

Comment:
commons-codec, which is one of our dependencies, has a method, `DigestUtils.sha1Hex(string)`

Reply:
Got it. Thank you.